src/containers/App.tsx

```diff
@@ -17,7 +18,16 @@ function App() {
     <ThemeProvider theme={Theme}>
       <Switch>
         <Route exact path="/events" component={HomeLandingPage} />
-        <Route exact path="/events/new" component={EventCreationPage} />
```

Comment:
i think you accidentally removed this line after adding your 2 new routes - we still need this for `EventCreationPage`

Reply:
It's still there on my end. I'll push again and see if that fixes it.